src/components/lyne-image/lyne-image.tsx

```diff
@@ -0,0 +1,366 @@
+   * Pass in a floating number between 0 (left) and 1 (right).
+   */
+  @Prop() public focalPointX: InterfaceImageAttributes['focalPointX'] = 1;
+
+  /**
+   * Pass in a floating number between 0 (top) and 1 (bottom).
+   */
+  @Prop() public focalPointY: InterfaceImageAttributes['focalPointY'] = 1;
+
+  /**
+   * In cases when the image is just serving a decorative purpose,
+   * we can hide it from assisitive technologies (e.g. an image
+   * in a teaser card)
+   */
+  @Prop() public hideFromScreenreader: InterfaceImageAttributes['hideFromScreenreader'] = false;
+
+  @Prop() public imageSrc?: string;
+
+  @Prop() public imageSrcExamples!: string;
+
+  /**
+   * With the support of native image lazy loading, we can now
+   * decide whether we want to load the image immediately or only
+   * once it is close to the visible viewport. The value eager is
+   * best used for images within the initial viewport. We want to
+   * load these images as fast as possible to improve the Core Web
+   * Vitals values. lazy works best for those images which are
+   * further down the page or invisible during the loading of the
+   * initial viewport.
+   */
+  @Prop() public loading: InterfaceImageAttributes['loading'] = 'eager';
+
+  /**
+   * With performance.mark you can log a timestamp associated with
+   * the name you define in performanceMark when a certain event is
+   * happening. In our case we will log the performance.mark into
+   * the PerformanceEntry API once the image is fully loaded.
+   * Performance monitoring tools like SpeedCurve or Lighthouse are
+   * then able to grab these entries from the PerformanceEntry API
+   * and give us additional information and insights about our page
+   * loading behaviour. We are then also able to montior these
+   * values over a long time period to see if our performance
+   * increases or decreases. Best to use lowercase strings here,
+   * separate words with underscores or dashes.
+   */
+  @Prop() public performanceMark?: string;
+
+  /**
+   * With the pictureSizesConfig object, you can pass in information
+   * into lyne-image about what kind of source elements should get
+   * rendered. mediaQueries accepts multiple Media Query entries
+   * which can get combined by defining a conditionOperator.
+   */
+  @Prop() public pictureSizesConfig?: any;
+
+  private _addLoadedClass(): void {
+    this._figureElement.classList.add('lyne-image__figure--loaded');
+  }
+
+  private _logPerformanceMarks(): void {
+    if (window.performance.mark && this.performanceMark) {
+      performance.clearMarks(this.performanceMark);
+      performance.mark(this.performanceMark);
+    }
+  }
+
+  private _matchMediaQueryDesignToken(breakpointSizeName): string {
+    const breakpointSizeNameValue = LyneDesignTokens[breakpointSizeName];
+
+    return `${breakpointSizeNameValue / LyneDesignTokens.TypoScaleDefault}rem`;
+
+  }
+
+  public render(): JSX.Element {
+
+    const attributes: InterfaceImageAttributes = {};
```

Comment:
After removing the custom type interface for `aria-hidden` and `role`, you can define the type for this object like this:

```
const attributes: {
  ariaHidden?: string;
  role?: string;
} = {};
```